### ISSUE TITLE
[BUG] Import of messages/index.json doesn't do anything

### ISSUE THREAD

**Issue by teisseire117** [victornpb/undiscord#250]
> WARNING: Github Issues are publicly visible, do not post sensitive information here.
> You may want to use the "Screenshot mode" before taking screenshots or copying logs.

**Describe the bug**
Cannot import messages/index.json after editing the file. Neither can you import it after importing the unedited file => then remove the channel IDs entries from the Undiscord console => then try to re-import the same unedited file(basically you can't import the same file twice).

The most problematic is not being able to import the file if you edited it first with notepad++. And I made sure I did it correctly. I even tried just changing one channel ID from the file to make sure the issue is not because I didn't edit it properly.

Basically, it only works if you extract the index.json from the archive Discord sent you, then immediately import it on UndiscorD.
You cannot edit the file.
And you cannot import it twice.
Otherwise it doesn't work
**Screenshots / Logs**

**System:**
 - OS: Windows 10
 - Browser: Chrome 92

**Comment by teisseire117** [comment]
edit:
I feel like I jinxed it, now it doesn't work at all. Even when I re-extract the file from the archive.

**Comment by Zinibow** [comment]
I have had this same issue, I just copied the index folder, and moved it to my downloads, then opened that. If that doesn't work try just uploading a plain text file or something, it will work eventually

**Comment by OSTurtle** [comment]
I have the same problem
